modules/azure/api_management_api/main.tf

```
@@ -185,6 +185,13 @@ resource "azurerm_api_management_api_policy" "api_policy" {
     %{endif}
   </inbound>
 
+  <backend>
+    <base />
+    %{if var.custom_backend_policy != null}
+      ${var.custom_backend_policy}
+      %{endif}
+  </backend>
+
```

I really have no idea how this works, but theoretically wouldn't the following be better?

```suggestion
  %{if var.custom_backend_policy != null}
    <backend>
      <base />
      ${var.custom_backend_policy}
    </backend>
  %{endif}
```

---

Please don't! This has a different meaning.

---

Well, these two options depend if we want to append the policy to the one in APIM or replace it. I am fine with either of these for the current case (at the moment, there's no backend policy in the APIM module, so the result will be the same, but it will be different when someone adds something there)